### ISSUE TITLE
Tab managment posibility 

### ISSUE THREAD

**Issue by ed-fruty** [philc/vimium#3170]
Does any posibility exists now for tabs managment ?
For example I press `Shift + t`, vomnibar with tabs opens. Can I close the tab from the list without going to that page ? Can it be some shortcut to close selected one ? Or maybe I can configure it now with keymaps ? 
Or for example I can filter tabs by typing "google" and want to close all filtered tabs for "google" keyword.

And maybe it would be pretty cool to add favicons for pages in vomnibar panel.

**Comment by gdh1995** [comment]
just an advertisement: you may have a try at my Vimium C (https://chrome.google.com/webstore/detail/vimium-c/hfjbmagddngcpeloejdejnfgbamkjaeg). It's open sourced on https://github.com/gdh1995/vimium-c . It supports that you use `Shift+Delete` to remove opened tabs and history items, just like what can be done on Chrome's omnibox. Favicons are also supported.

**Comment by ed-fruty** [comment]
@gdh1995 looks nice. But As I see now, it doesn't support `live search`, like original `Vimium`

**Comment by gdh1995** [comment]
Yes, it's mainly because I don't want my Vimium-C to visit the Internet frequently...

**Comment by ed-fruty** [comment]
@gdh1995 , can you make that optional and configurable ?)

**Comment by gdh1995** [comment]
Sorry but in the next months I have no enough time to re-add this feature.

So there's indeed a requirement that Vimium implements `Shift+Enter` to remove tabs.

**Comment by ed-fruty** [comment]
Up ;)

**Comment by balta2ar** [comment]
@gdh1995 Could you please publish your extension to Mozilla addons store?

**Comment by gdh1995** [comment]
Um, it's in my development plan, but there's no certain due date (about one month~).
